package.json

```diff
@@ -60,18 +61,18 @@
     "parser": "babel-eslint"
   },
   "scripts": {
-    "lint": "node node_modules/standard/bin/cmd.js src/**/*.js",
```

Comment:
I think all versions of npm support it, but I'm not 100% sure. `node_modules` gets prioritised over global. More info [here](https://docs.npmjs.com/cli/run-script).